crates/cargo-wdk/src/actions/build/mod.rs

```diff
@@ -223,47 +228,47 @@ impl<'a> BuildAction<'a> {
             absolute(cargo_metadata.workspace_root.as_std_path()).map_err(|e| {
                 BuildActionError::NotAbsolute(cargo_metadata.workspace_root.clone().into(), e)
             })?;
-        let target_directory =
-            absolute(cargo_metadata.target_directory.as_std_path()).map_err(|e| {
-                BuildActionError::NotAbsolute(cargo_metadata.target_directory.clone().into(), e)
-            })?;
         if workspace_root.eq(&working_dir) {
             // If the working directory is root of a standalone project or a
             // workspace
             debug!(
                 "Running from standalone project or from a root of a workspace: {}",
                 working_dir.display()
             );
-            let mut failed_atleast_one_workspace_member = false;
-            for package in workspace_packages {
-                let package_root_path: PathBuf = package
-                    .manifest_path
-                    .parent()
-                    .expect("Unable to find package path from Cargo manifest path")
-                    .into();
+            let failed_atleast_one_workspace_member =
+                workspace_packages
+                    .into_iter()
+                    .try_fold(false, |failed_any, package| {
+                        let package_root: PathBuf = package
+                            .manifest_path
+                            .parent()
+                            .expect("Unable to find package path from Cargo manifest path")
+                            .into();
+
+                        let package_root = absolute(package_root.as_path())
+                            .map_err(|e| BuildActionError::NotAbsolute(package_root.clone(), e))?;
+                        debug!(
+                            "Building workspace member package: {}",
+                            package_root.display()
+                        );
+
+                        match self.build_and_package(
+                            &package_root,
+                            wdk_metadata.as_ref().ok(),
+                            package,
+                        ) {
+                            Ok(()) => Ok::<bool, BuildActionError>(failed_any),
+                            Err(e) => {
+                                err!(
+                                    "Error building the workspace member project: {}, error: {:?}",
+                                    package_root.display(),
+                                    anyhow::Error::new(e)
+                                );
+                                Ok::<bool, BuildActionError>(true)
+                            }
+                        }
+                    })?;
```

Comment:
This seems like a completely unnecessary change. Please do not make drive-by changes for no reason. Better to stick to changes related to the goal of the PR.